media/css/pebbles/components/_survey-message.scss

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#survey-message {
+    @include border-box;
+    background: #fff0b3;
+    bottom: 0;
+    min-height: 30px;
+    overflow: hidden;
+    padding: 10px 20px;
+    position: fixed;
+    text-align: center;
+    transform: translateY(100%);
```

Comment:
Yeah, it was more for consistency than compatibility. I'm not especially concerned about catering to 5-year-old webkits. 👍